PowerValidatedSolutions.psm1

```diff
@@ -24802,6 +24854,68 @@ Function Start-vRSLCMProductNode {
 }
 Export-ModuleMember -Function Start-vRSLCMProductNode
 
+Function Get-vRSLCMPSPack {
```

Comment:
Do you want to update function details to "Product Support Packs" in these two functions to align with the user functions?

Reply:
Good call